#!/usr/bin/env python
PACKAGE = "hector_change_map"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("current_robot_layer", int_t, 0, "current robot layer", 0, 0, 10)

exit(gen.generate(PACKAGE, "hector_change_map_node", "HectorChangeMap"))
